graph/patterns/dictionary.md

```diff
@@ -36,8 +36,9 @@ Because dictionary entries are removed by setting the value to `null`, dictionar
 Open questions:
 
 - Can/should PUT be supported on the dictionary property and/or the entry value?
-- What does OData say about being able to POST to a structured property? Will OData Web API allow that?
 - Must an implementer support PATCH at both the dictionary level and the entry level?
+- Should we also allow DELETE to a property to be equivalent to setting to null?
+- Why do we not allow mixed primitives or mixed primitive/complex typed values? what about collections?
```

Comment:
Do we have a constraint here for SDKs or AGS?  Otherwise, why wouldn't we allow this?
If we do, should we have an updated example to show this usage?

I also wonder whether we should have 2 examples - 1 simple example that replaces the standard KeyValue(String primitive) pair which is the canonical example, and something more complex with mixed primitives/complex types/collections.